src/viadot/orchestration/prefect/tasks/bcp.py

```diff
@@ -64,8 +70,33 @@ def bcp(
     elif on_error == "fail":
         max_error = 1
     else:
-        msg = "Please provide correct 'on_error' parameter value - 'skip' or 'fail'."
+        msg = "Please provide correct 'on_error' parameter value - 'skip' or 'fail'. "
         raise ValueError(msg)
+    bcp_command = [
+        "/opt/mssql-tools/bin/bcp",
+        fqn,
+        "in",
+        path,
+        "-S",
+        server,
+        "-d",
+        db_name,
+        "-U",
+        uid,
+        "-P",
+        pwd,
+        "-b",
+        str(chunksize),
+        "-m",
+        str(max_error),
+        "-c",
+        "-v",
```

Comment:
So the only difference is the `-v` flag, which according to docs, shows the version of the tool? 😅 https://learn.microsoft.com/en-us/sql/tools/bcp-utility?view=sql-server-ver16&tabs=windows#-v

Also, why replace `shell_run_command()` with `subprocess.run()`?

Reply:
I don't know why but `shell_run_command()` didn't work for this command

Reply:
Also in this case casting int parameters as str was needed